sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -57,9 +57,8 @@ public partial class CredentialUnavailableException : Azure.Identity.Authenticat
     }
     public partial class DefaultAzureCredential : Azure.Core.TokenCredential
     {
-        public DefaultAzureCredential() { }
         public DefaultAzureCredential(Azure.Identity.DefaultAzureCredentialOptions options) { }
-        public DefaultAzureCredential(bool includeInteractiveCredentials) { }
+        public DefaultAzureCredential(bool includeInteractiveCredentials = false) { }
```

Comment:
I believe you were planning to revert this change no?

Reply:
This is reverting the change, the change itself was part of another PR that has been merged already.

Reply:
sorry, I read it backwards :smile:.